op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -303,6 +273,110 @@ func planCall(t devtest.T, call txintent.Call) txplan.Option {
 	return txplan.Combine(plan...)
 }
 
+func setupLoadTest(gt *testing.T) (devtest.T, *L2, *L2) {
+	if testing.Short() {
+		gt.Skip("skipping load test in short mode")
+	}
+	t := devtest.SerialT(gt)
+	l2A, l2B := setupL2s(t)
+	collectMetrics(t, l2A.BlockTime)
+	return t, l2A, l2B
+}
+
+func setupL2s(t devtest.T) (*L2, *L2) {
+	sys := presets.NewSimpleInterop(t)
+	blockTimeA := time.Duration(sys.L2ChainA.Escape().RollupConfig().BlockTime) * time.Second
+	blockTimeB := time.Duration(sys.L2ChainB.Escape().RollupConfig().BlockTime) * time.Second
+	t.Require().Equal(blockTimeA, blockTimeB)
+	blockTime := blockTimeA
+
+	budget := eth.OneEther
+	if budgetStr, exists := os.LookupEnv("NAT_INTEROP_LOADTEST_BUDGET"); exists {
+		amount, err := strconv.ParseUint(budgetStr, 10, 64)
+		t.Require().NoError(err)
+		budget = eth.Ether(amount)
+	}
+	l2ELA := sys.L2ChainA.PublicRPC()
+	l2ELB := sys.L2ChainB.PublicRPC()
+	funderA := dsl.NewFunder(sys.Wallet, sys.FaucetA, l2ELA)
+	funderB := dsl.NewFunder(sys.Wallet, sys.FaucetB, l2ELB)
+	const numEOAsPerChain = 1
```

Comment:
Lowering the number of EOAs to 1 so bursts tests are faster. I'll add a todo with an issue for the Burst problems discussed over Slack.